src/isomorphic/classic/class/__tests__/ReactClassMixin-test.js

```diff
@@ -368,24 +372,24 @@ describe('ReactClass-mixin', function() {
   });
 
   it('should include the mixin keys in even if their values are falsy',
-    function() {
-      var mixin = {
-        keyWithNullValue: null,
-        randomCounter: 0
-      };
+      function() {
```

Comment:
There's other tests where we're breaking lines like this, so I think it's fine.

Is trying to hit 80 columns everywhere a goal or not? Currently it's making the linter incredibly noisy. (235 warnings)